frontend/pages/hosts/details/cards/Software/HostSoftware.tsx

```diff
@@ -30,14 +30,17 @@ import SoftwareFiltersModal from "pages/SoftwarePage/components/SoftwareFiltersM
 import {
   buildSoftwareFilterQueryParams,
   buildSoftwareVulnFiltersQueryParams,
-  getSoftwareFilterFromQueryParams,
   getSoftwareVulnFiltersFromQueryParams,
   ISoftwareVulnFiltersParams,
 } from "pages/SoftwarePage/SoftwareTitles/SoftwareTable/helpers";
 import { generateSoftwareTableHeaders as generateHostSoftwareTableConfig } from "./HostSoftwareTableConfig";
 import { generateSoftwareTableHeaders as generateDeviceSoftwareTableConfig } from "./DeviceSoftwareTableConfig";
 import HostSoftwareTable from "./HostSoftwareTable";
-import { getInstallErrorMessage, getUninstallErrorMessage } from "./helpers";
+import {
+  getHostSoftwareFilterFromQueryParams,
+  getInstallErrorMessage,
```

Comment:
Why are these imports new?

Reply:
@iansltx 
Just `getHostSoftwareFilterFromQueryParams` is new from `"./helpers"`. Originally was using `getSoftwareFilterFromQueryParams` from `"pages/SoftwarePage/SoftwareTitles/SoftwareTable/helpers"` which the typing mismatched (string v. boolean) hence the bug so created a new `getHostSoftwareFilterFromQueryParams` to put in local `helpers`.

Reply:
...yep, sorry. Reading comprehension issue on my part.

Reply:
nw, thanks for the swift review <3